networking/p2p/src/main/java/tech/pegasys/teku/networking/p2p/libp2p/gossip/GossipHandler.java

```diff
@@ -64,7 +64,9 @@ public GossipHandler(
           GOSSIP_MAX_SIZE);
       return VALIDATION_FAILED;
     }
-    Bytes bytes = Bytes.wrapByteBuf(message.getData()).copy();
```

Comment:
What's wrong with the existing line?

Reply:
Mentioned the `Bytes.wrapByteBuf` bug above